openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/logging/console-scraper.service.ts

```diff
@@ -48,14 +48,16 @@ export class ConsoleScraper {
     }
 
     private _handleLogMessage(observer: Subscriber<ConsoleMessage>, level: LogLevel, ...args: any[]) {
-        let message = '';
-
-        if (args.length <= 0) {
+        if (!args || args.length <= 0) {
             return;
         }
 
+        let message = '';
+
         args.forEach(arg => {
-            if (typeof arg == 'object') {
+            if (!arg) {
+                // ignore falsey arguments
+            } else if (typeof arg == 'object') {
```

Comment:
since that if isn't doing anything could this just be changed to 
```suggestion
       if (arg && typeof arg === 'object') {
```

Reply:
I don't want this to fall into the `else` condition either.

Reply:
Okay then I'd suggest this rather than have the blank if: 
```suggestion
        args.forEach(arg => {
            if (arg !== null || arg !== undefined) {
               if (typeof arg === 'object') {
                  let stringify = this._jsonStringifySanitize(arg);

                  if (stringify.length > 0) {
                      if (message.length > 0) {
                          message += '; ';
                      }
                      message += stringify;
                  }
               } else {
                  if (message.length > 0) {
                      message += '; ';
                  }
                  message += arg;
               }
           }
        });
```